t/ui/13-admin.t

```diff
@@ -30,18 +30,25 @@ eval 'use Test::More::Color';
 eval 'use Test::More::Color "foreground"';
 
 use File::Path qw/make_path remove_tree/;
+use Module::Load::Conditional qw/can_load/;
 
 my $test_case = OpenQA::Test::Case->new;
 $test_case->init_data;
 
 use t::ui::PhantomTest;
 
-my $driver = t::ui::PhantomTest::call_phantom();
-if ($driver) {
+# skip if phantomjs or Selenium::Remote::WDKeys isn't available
+# we set $enterkey here to avoid using an undefined bareword later if
+# WDKeys (and hence the KEYS const) is not available.
+my $driver   = t::ui::PhantomTest::call_phantom();
+my $enterkey = '';
+if ($driver && can_load(modules => {'Selenium::Remote::WDKeys' => undef,})) {
```

Comment:
Won't that fail in the case that the module isn't available? That's the reason this `$enterkey` dodge exists. I think perl will just consider `Selenium::Remote::WDKeys` to be a bareword and explode. Even though the relevant code would never be *run* on the "no Selenium" path, it gets evaluated at compile time and causes an error.